src/sas/qtgui/Perspectives/Invariant/InvariantDetails.py

```diff
@@ -202,4 +205,7 @@ def checkValues(self):
             msg += "The sum of all extrapolated contributions is higher " \
                    "than 5% of the invariant.\n"
 
+        if msg == '':
```

Comment:
Do we want to output it or it was set for debugging purposes?

Reply:
> Do we want to output it or it was set for debugging purposes?

No. That can stay in. Similar messaging is in 4.x.

Reply:
Yes this was very deliberate (and took me a while to figure out). Currently, without this, the default is message in the details warnings box is: `No Details on calculations available.`  Which is misleading since the implication to me (and confused me when I was using it at first) is that something went wrong.  

NOTE: this is set in the panel initialization as the default entry at which point it is true that no details are avalialbe since the code running the calculation has not run so decided that changing it there was not correct?

Also, oddly line 59 sets the same message but is never used anywhere that I can see! (and changing it has no effect I could see).  Was going to delete but decided I didn't know enough?  Should we delete?
```python
        self.warning_msg = "No Details on calculations available...\n"
```